main_test.go

```diff
@@ -33,23 +34,9 @@ var testErr = errors.New("test")
 func Test_getK8sManagerOptions(t *testing.T) {
 
 	t.Run("successfully get k8s manager options", func(t *testing.T) {
-		exiter := &mockExiter{}
 		t.Setenv("NAMESPACE", "default")
```

Comment:
instantiate the config with the default namespace and do an assertEqual() with the returned options